lib/cc/analyzer/issue_validations/location_format_validation.rb

```diff
@@ -2,42 +2,50 @@ module CC
   module Analyzer
     module IssueValidations
       class LocationFormatValidation < Validation
-        def valid?
-          if location["lines"]
-            valid_lines?(location["lines"])
-          elsif location["positions"]
-            valid_positions?(location["positions"])
-          else
-            false
+        class Validator
```

Comment:
The reason for extracting the class is so we can use the validator in a context where we have access to the location, but not the issue (like in `SourceExtractor`). It would've been cool to just use the existing validator as-is, but it requires having access to the issue... agreed this feels a little weird